Blueprints/BlueprintDefinitions/vs2022/SimpleDynamoDBFunction-FSharp/template/test/BlueprintBaseName.1.Tests/FunctionTest.fs

```diff
@@ -1,12 +1,9 @@
 ﻿namespace BlueprintBaseName._1.Tests
```

Comment:
This file has significant changes to deal with change to move to new major version of Amazon.Lambda.DynamoDBEvents which removed the dependency on the SDK.